<launch>

<arg name="path_slam_cofig" default = "/home/yipuzhao/ros_workspace/package_dir/ORB_Data/"/>
<arg name="path_data_logging" default = "/mnt/DATA/tmp/ClosedNav/debug/"/>
<arg name="num_all_feature" default = "1600" />
<arg name="do_rectify" default = "true" />
<arg name="do_vis" default = "false" />


<!-- call baselin ORB-SLAM for vision-based state estimation  -->
<node pkg="gf_orb_slam2" type="Stereo" name="visual_slam" args="$(arg path_slam_cofig)/ORBvoc.bin $(arg path_slam_cofig)/Gazebo_yaml/pinhole_stereo.yaml $(arg num_all_feature) $(arg do_rectify) $(arg do_vis) /multisense_sl/camera/left/image_raw /multisense_sl/camera/right/image_raw $(arg path_data_logging)">
</node>



</launch>
